src/components/StapledSidebar/StapledSidebar.style.tsx

```diff
@@ -50,12 +59,27 @@ export const StyledSidebarLink = styled.a`
   font-weight: 500;
 `;
 
+const NAV_HEIGHT = 65;
+const OUTER_PADDING = 16;
+const INNER_PADDING = 40;
 export const SectionHeader = styled(Typography)<{
   variant: String;
   component: String;
 }>`
-  padding-top: 105px;
-  margin-top: -65px;
+  padding-top: ${NAV_HEIGHT + INNER_PADDING - 1}px;
+  margin-top: -${NAV_HEIGHT - OUTER_PADDING}px;
+
+  &:before {
+    display: block;
+    width: 100%;
+    content: ' ';
+    transform: translateY(-${INNER_PADDING}px);
+    border-top: 1px solid rgba(0, 0, 0, 0.12);
+  }
```

Comment:
Wowsers...  I'm not sure exactly what's going on here (my CSS is weak), but it seems pretty complicated for a section heading...  Consider adding a comment (optional)?

Reply:
Comment added! this positions the scroll-to links perfectly aligned with the headers.